From a0543d7f3de6fa0c088868e91400e3ada4198238 Mon Sep 17 00:00:00 2001
From: fupan li <fupan.li@windriver.com>
Date: Fri, 17 Jan 2014 02:07:14 -0800
Subject: [PATCH] atomisp: fix compilation build error

arch/x86/platform/intel-atom/platform_camera.c:174:3: error: implicit declaration of function 'kzalloc'
arch/x86/platform/intel-atom/platform_camera.c:186:3: error: implicit declaration of function 'kfree' [-Werror=implicit-function-declaration]

Signed-off-by: fupan li <fupan.li@windriver.com>
---
 arch/x86/platform/intel-atom/platform_camera.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/x86/platform/intel-atom/platform_camera.c b/arch/x86/platform/intel-atom/platform_camera.c
index b7301ce..5651921 100644
--- a/arch/x86/platform/intel-atom/platform_camera.c
+++ b/arch/x86/platform/intel-atom/platform_camera.c
@@ -15,6 +15,7 @@
 #include <linux/module.h>
 #include <linux/atomisp_platform.h>
 #include <media/v4l2-subdev.h>
+#include <linux/slab.h>
 #include "platform_camera.h"
 #include "platform_mt9m114.h"
 #include "platform_ov5640_1.h"
-- 
1.7.5.4

